### DISCUSSION TITLE
N(EAP_ONLY) and N(MSG_ID_SYN_SUP)

### DISCUSSION THREAD

**Discussion by grahamqs** [strongswan/strongswan#1119] [Q&A]
Hi

Does anyone know why a EAP_ONLY notify payload if sent, if I'm just using pubkey authentication ? Is there a way to disable it ?

Also does anyone know how I can disable N(MSG_ID_SYN_SUP) ?

many thanks

**Comment by Thermi** [comment]
It just indicates support for EAP only authentication in the daemon. It does not indicate intend to use it.
